packages/terra-search-field/src/terra-dev-site/doc/example/SearchFieldProgrammaticSet.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import Button from 'terra-button/lib/Button';
+import SearchField from '../../../SearchField';
+import styles from '../../test/search-field/examplestyles.scss';
+
+const cx = classNames.bind(styles);
+
+class SearchFieldProgrammaticSet extends React.Component {
```

Comment:
Not a big deal, but we should move away from class components and use functional components with hooks, whenever possible.

Reply:
Fixed here https://github.com/cerner/terra-core/pull/2800/commits/63069a3151a47224cf7d2c3c4ae05354b364304e#diff-5b54d3bd01a4e6076cc428ee0d369f6cR1